Commands/Commands.cs

```diff
@@ -23,14 +24,21 @@ public async Task GetLatency(CommandContext ctx)
                 .Build();
             await ctx.Channel.SendMessageAsync(embed: latency);
         }
-        [Command("F")]
-        private async Task F()
-        {
-            await ReplyAsync("F");
-        }
-        [Command("say")]
-        [Summary("say what user gave as input")]
-        public Task SayAsync([Remainder] [Summary("What shoukd the bot say")] string echo)
-            => ReplyAsync(echo);
+
+        //F command
+        [Command("F"), Aliases("f")]
```

Comment:
There is no need for that alias, commands are not case sensitive.

Reply:
Ok